src/patternfly/components/Splitter/splitter.scss

```diff
@@ -0,0 +1,70 @@
+.pf-c-splitter {
+  // splitter
+  --pf-c-splitter--Height: var(--pf-global--spacer--sm);
+  --pf-c-splitter--Width: 100%;
+  --pf-c-splitter--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-splitter--Cursor: row-resize;
+  --pf-c-splitter--m-vertical--Height: 100%;
+  --pf-c-splitter--m-vertical--Width: var(--pf-global--spacer--sm);
+  --pf-c-splitter--m-vertical--Cursor: col-resize;
+
+  // splitter handle
+  --pf-c-splitter__handle--Top: 0;
+  --pf-c-splitter__handle--Right: 0;
+  --pf-c-splitter__handle--Bottom: 0;
+  --pf-c-splitter__handle--Left: 50%;
+  --pf-c-splitter--m-vertical__handle--Top: 50%;
+  --pf-c-splitter--m-vertical__handle--Right: 0;
+  --pf-c-splitter--m-vertical__handle--Left: 0;
+  --pf-c-splitter--m-vertical__handle--Bottom: 0;
+
+  // splitter handle line
+  --pf-c-splitter__handle-line--BackgroundColor: var(--pf-global--Color--100);
+  --pf-c-splitter__handle-line--Height: #{pf-size-prem(1px)};
+  --pf-c-splitter__handle-line--Width: #{pf-size-prem(12px)};
+  --pf-c-splitter--m-vertical__handle-line--Height: #{pf-size-prem(12px)};
+  --pf-c-splitter--m-vertical__handle-line--Width: #{pf-size-prem(1px)};
+
+  position: relative;
```

Comment:
I think, in use, this will need to be absolutely positioned. Then `top`, `bottom`,`left`, and `right` would control where it appears in the component?

Reply:
Could be... though I think the component should style it because it may not be absolutely positioned. You could end up creating a container for the splitter instead. That might be a better approach so that you could do something like change `--pf-c-splitter--Width` and not need to make any sort of padding or other adjustments in the component that includes it, which could be tricky since `.pf-c-drawer` (as an example) can't read `--pf-c-splitter--Width`, and automatically create space for the splitter size.

Reply:
yep, i was thinking the layout would handle that - or should i make this `position: absolute`

Reply:
I'm in favor of allowing the context it's used to define the element's positioning, and is inline with how the rest of our components are positioned used in another component.